### PR TITLE
enable rsync of oncokb annotated maf for impact

### PR DESCRIPTION
**Pull request by sheridancbio**
enable rsync of oncokb annotated maf for impact
- general script to create an index.html file (create_web_directory_index_file.sh)
- function which rsyncs the oncokb annotation files and creates/rsyncs the index.html file (rsync_oncokb_annotation_files.sh) added to oncokb-annotator script
- the rsync script uses an identity file for ssh-based transfer

### CODE REVIEW

**Comment by sheridancbio** [comment]
@ao508 @averyniceday @mandawilson @n1zea144 : I have found out how we can transfer files from the cbioportal_importer account directly, so I am now going to combine the rsync script into the others.

**Comment by sheridancbio** [comment]
@ao508 @averyniceday now the prepare script has been merged into the rsync script and there is no trigger file needed. An ssh identity file is used to allow the rsync to ramen to occur.
Any further comments / requests for updates are welcome.

File: import-scripts/oncokb-annotator.sh [hunk 1]
```diff
@@ -149,9 +149,8 @@ if [ $ONCOKB_ANNOTATION_SUCCESS -eq 0 ] ; then
     message="OncoKB Annotation failed for MSKSOLIDHEME. Check logs for more details."
     send_failure_messages "$message" "MSKSOLIDHEME OncoKB Annotation Failure" "MSKSOLIDHEME OncoKB Annotation"
 else
-    echo "committing OncoKB Annotation for MSKSOLIDHEME"
-    cd $MSK_SOLID_HEME_DATA_HOME ; $GIT_BINARY add * ; $GIT_BINARY commit -m "Latest MSKSOLIDHEME OncoKB Annotations"
-    $GIT_BINARY push origin
+    echo "Triggering rsync to download.cbioportal.org for OncoKB Annotation for MSKSOLIDHEME"
+    $PORTAL_HOME/scripts/prepare_oncokb_annotation_rsync.sh
```
Comment (averyniceday) [inline import-scripts/oncokb-annotator.sh]: does this script still exist?

Comment (sheridancbio) [reply import-scripts/oncokb-annotator.sh]: you are right .. I will fix this now

**Pull request merged**